server/index.js

```diff
@@ -1,12 +1,68 @@
+
+/* eslint-disable no-console */
 require('dotenv/config');
 const express = require('express');
 const errorMiddleware = require('./error-middleware');
 const staticMiddleware = require('./static-middleware');
+const fetch = require('node-fetch');
+const ClientError = require('./client-error');
+const cors = require('cors');
+// const { query } = require('express');
 
 const app = express();
+app.use(cors());
+
+// eslint-disable-next-line no-unused-vars
+const formatData = json => {
+  // console.log(JSON.stringify(json));
+  return json.response.venues
+    .map(({ id, location, name, categories }) => {
+      const { address } = location;
+      const { pluralName } = categories[0];
+
+      return {
+        id,
+        address,
+        name,
+        pluralName
+      };
+    });
+};
 
 app.use(staticMiddleware);
 
+app.get('/api/venue/image/:id?', (req, res, next) => {
+  const id = req.params.id;
+
+  fetch(`https://api.foursquare.com/v2/venues/${id}/photos?client_id=EZOAI304JHK222F243HNN4KULBKUGKG10DPYVDWCHNJWB3HT&client_secret=IG3AUD5AEZ1TVO5PMT43BYTL2WRTIRSKWBUFQ4HZAVTZJYI3&v=20210514&v=20210514`)
```

Comment:
You should put your id and secret into environment variables and use those instead. Your secret info is now publicly visible for anybody to steal 😱